test/mailers/previews/contact_mailer_preview.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/contact_mailer
+class ContactMailerPreview < ActionMailer::Preview
+  def send_information
+    ContactMailer.send_information(
+      Contact.create(
+        name: "Peter",
+        email: "Enis@at.de",
+        message: "Ja, lol bin ich schon drin so schnell ging das ich bin im Internet"
+      ))
+  end
+
+  def send_to_management
+    ContactMailer.send_to_management(
+      Contact.create(
+        name: "Peter",
+        email: "Enis@at.de",
+        message: "Ja, lol bin ich schon drin so schnell ging das ich bin im Internet"
+      ),
+      User.create(
```

Comment:
Nutze nach Möglichkeit die [Rails-Fixtures](https://edgeguides.rubyonrails.org/testing.html#the-low-down-on-fixtures) und erzeuge keine weiteren Daten in der DB, die du vielleicht gar nicht benötigst. Jedes `create` macht deine Tests viel langsamer und bei tausenden Tests in der Anwendung spielt das eine Rolle.

Reply:
Alles klar